haskell/test/Test/Ligo/BaseDAO/Common.hs

```diff
@@ -203,7 +203,7 @@ originateLigoDaoWithBalance extra config balFunc = do
         , (#owner .! owner2, #operator .! operator2, #token_id .! unfrozenTokens)
         ]
 
-  now <- getNow
+  now <- getLevel
```

Comment:
Likewise in these uses of the word `now`

Reply:
Changed